MoneySplitter.Services/Defines.cs

```diff
@@ -32,6 +32,7 @@ public static class Friends
                 public const string ADD_FRIEND = "/add";
                 public const string ALL_FRIENDS= "/all";
                 public const string REMOVE_FRIEND = "/remove";
+                public const string FRIEND_ID = "?friendId=";
```

Comment:
you can make method like CreateQueryParameter(string parameterName, parameterValue) and return $"?{parameterName}={parameterValue}" to avoid dublicates of "?name=" strings

Reply:
fixed